pyanaconda/modules/storage/checker/utils.py

```diff
@@ -429,15 +431,30 @@ def verify_mounted_partitions(storage, constraints, report_error, report_warning
         if not disk.partitioned:
             continue
 
-        for part in disk.format.partitions:
-            part_dev = storage.devicetree.get_device_by_path(part.path)
+        already_checked_partitions = set()
```

Comment:
It might be easier to first collect a set of partitions that need to be checked and then check these partitions in another loop. This solution is a little hard to follow.

Reply:
Done...